### PR TITLE
add caution about removing package

### PR DESCRIPTION
**Pull request by anatawa12**

### CODE REVIEW

File: Docs/docs/guides/create-listing.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 
+:::caution
+
+You SHOULD NOT remove any versions you've published.
+It breaks projects under source control system.
+Projects under source control system doesn't include contents of vpm package.
+See [Using Source Control with the VPM](https://vcc.docs.vrchat.com/vpm/source-control) for more details packages on source control system.
+
+:::
+
```
Comment (Faxmashine) [inline Docs/docs/guides/create-listing.md:13]: What do you think of shortening the warning?

```suggestion
:::caution

Do not remove old versions of your VPM package when adding new versions.
Removing old versions will break projects using [source control](https://vcc.docs.vrchat.com/vpm/source-control).

:::

```

**Review by Faxmashine — CHANGES_REQUESTED** [review]
I wonder if this warning should also be shown at https://vcc.docs.vrchat.com/vpm/packages#versions-and-ranges . The person making the listing might not be the same person who made the package.

File: Docs/docs/guides/create-listing.md (hunk 1 above)
Comment (anatawa12) [reply Docs/docs/guides/create-listing.md:13]: I think shortening the warning is good. I'll copy notice to packages soon.

File: Docs/docs/vpm/packages.md [hunk 2]
```diff
@@ -94,3 +94,12 @@ We've improved upon the functionality provided by the Unity Package Manager by s
 For example, we use the format "^3.1.x" in many of the official templates and VRC SDK packages. This format is designed to match any version past 3.1.0, but it restricts changes to the major version number. This is because a change in the major version number signifies breaking changes that may not be compatible with previous versions. For instance, if a package has version 3.5.2, it will match the "^3.1.x" format, whereas version 4.1.0 will not.
 
 This format also supports [Prerelease Packages](#prerelease-packages).
+
+<!-- Don't forget sync this caution with guides/create-listing.md -->
+
+:::caution
+
+Do not remove old versions of your VPM package when adding new versions.
```
Comment (anatawa12) [inline Docs/docs/vpm/packages.md:102]:
```suggestion
Do not remove old versions of your VPM packages.
```

I think when adding new versions is not required.

File: Docs/docs/guides/create-listing.md [hunk 3]
```diff
@@ -2,6 +2,15 @@
 
 You can create and share your own custom package listings, including your own packages as well as open source packages from the community.
 
+<!-- Don't forget sync this caution with vpm/packages.md -->
+
+:::caution
+
+Do not remove old versions of your VPM packages.
```
Comment (Faxmashine) [inline Docs/docs/guides/create-listing.md:9]:
```suggestion
Do not remove old versions of your VPM packages after publishing them.
```

File: Docs/docs/vpm/packages.md [hunk 4]
```diff
@@ -94,3 +94,12 @@ We've improved upon the functionality provided by the Unity Package Manager by s
 For example, we use the format "^3.1.x" in many of the official templates and VRC SDK packages. This format is designed to match any version past 3.1.0, but it restricts changes to the major version number. This is because a change in the major version number signifies breaking changes that may not be compatible with previous versions. For instance, if a package has version 3.5.2, it will match the "^3.1.x" format, whereas version 4.1.0 will not.
 
 This format also supports [Prerelease Packages](#prerelease-packages).
+
+<!-- Don't forget sync this caution with guides/create-listing.md -->
+
+:::caution
+
+Do not remove old versions of your VPM packages.
```
Comment (Faxmashine) [inline Docs/docs/vpm/packages.md:102]:
```suggestion
Do not remove old versions of your VPM packages after publishing them.
```

**Review by Faxmashine — CHANGES_REQUESTED** [review]
I think it might be best to remind the reader that this refers to the publishing of packages, and not packages in general.

'Do not remove old versions of your VPM packages' might imply that *users* can't remove old versions.

**Comment by anatawa12** [comment]
@Faxmashine It looks this change is not published yet. Could you let me know progress?

**Comment by Faxmashine** [comment]
Sorry for the delay! Your changes are live now.

Thank you for your contribution.

**Pull request merged**